examples/dm-thread-pagination.example.ts

```diff
@@ -0,0 +1,14 @@
+import { IgApiClient } from '../src';
+
+(async () => {
+  const ig = new IgApiClient();
+  ig.state.generateDevice(process.env.IG_USERNAME);
+  await ig.account.login(process.env.IG_USERNAME, process.env.IG_PASSWORD);
+  const inboxFeed = ig.feed.directInbox(10);
+  do {
+    let threads = await inboxFeed.items();
+    threads.forEach(element => {
+      console.log(element['users'][0]['username']);
```

Comment:
You could rename `element` to `thread`, because this element is effectively a thread. But you shouldn't access via string-literal so this would be something like `thread.users[0].username`.